templates/_tesseract_options.html.twig

```diff
@@ -0,0 +1,23 @@
+<fieldset id="tesseract-fields">
+    <legend>{{ msg('tesseract-options') }}</legend>
+    <div class="form-group">
+        <label for="psm">{{ msg('tesseract-psm-label') }}</label>
+        <select name="psm" id="psm" class="form-control">
+            {# Construct the array so that the default (3) comes first. #}
+            {% for i in 0..13 %}
+                {# Each i18n message is 'tesseract-psm-N' where N is the PSM value. #}
```

Comment:
Should we do the MediaWiki thing of listing out all messages in full, for easier grepping?

Reply:
Yes! Good idea.